Update win11-overview.md (#7366)
MicrosoftDocs/microsoft-365-docs: microsoft-365/managed-desktop/intro/win11-overview.md

For specific steps to follow to get Windows 11 installed on your Microsoft Managed Desktop devices, see [Preview and test Windows 11 with Microsoft Managed Desktop](../working-with-managed-desktop/test-win11-mmd.md).

## Timeline for Windows 10 and Windows 11

Windows 11 became generally available on October 4, 2021. It is ready for consumer and enterprise deployment and is a fully supported platform. We will begin scheduling deployments for all Microsoft Managed Desktop devices starting January 2023, but will provide full support for those that wish to deploy Windows 11 sooner. We'll consult and advise admins to develop and implement migration plans for each tenant based on technical readiness and your business considerations.

> Microsoft Managed Desktop continues to support Windows 10 in parallel until it reaches end of enterprise support. Please see [Windows 10 release information](/windows/release-health/release-information) for life cycle information.

Reviewer: ```suggestion
Microsoft Managed Desktop continues to support Windows 10 in parallel until it reaches end of enterprise support. See [Windows 10 release information](/windows/release-health/release-information) for life cycle information.
```

## Assessing pre-release versions of Windows 11

More than 95% of Microsoft Managed Desktop devices are eligible for Windows 11, so you might want to try the upgrade on test devices prior to production deployment. For more about Windows 11 system requirements, see [Windows 11 requirements](/windows/whats-new/windows-11-requirements). 

> For Microsoft Managed Desktop devices, you can [add devices to the Windows 11 test device group] (https://docs.microsoft.com/en-us/microsoft-365/managed-desktop/working-with-managed-desktop/test-win11-mmd?view=o365-worldwide#add-devices-to-the-windows-11-test-group). This group receives the Windows 11 general availability build along with a Microsoft Managed Desktop baseline configuration. Once added to the device group, allow 1-2 days for a device to pick up the new settings and be offered Windows 11.

Reviewer: ```suggestion
For Microsoft Managed Desktop devices, you can [add devices to the Windows 11 test device group](/microsoft-365/managed-desktop/working-with-managed-desktop/test-win11-mmd?view=o365-worldwide#add-devices-to-the-windows-11-test-group). This group receives the Windows 11 general availability build along with a Microsoft Managed Desktop baseline configuration. Once added to the device group, allow one to two days for a device to pick up the new settings and be offered Windows 11.
```

> For your devices that aren't managed by Microsoft Managed Desktop, you can read [Endpoint Manager guidance](https://techcommunity.microsoft.com/t5/microsoft-endpoint-manager-blog/endpoint-manager-simplifies-upgrades-to-windows-11/ba-p/2771886) to learn about deploying Windows 11 yourself. If you have devices running Windows 11 and later enroll them in Microsoft Managed Desktop, they won't revert back to Windows 10.

Reviewer: ```suggestion
For your devices that aren't managed by Microsoft Managed Desktop, you can read [Endpoint Manager guidance](https://techcommunity.microsoft.com/t5/microsoft-endpoint-manager-blog/endpoint-manager-simplifies-upgrades-to-windows-11/ba-p/2771886) to learn about deploying Windows 11 yourself. If you have devices running Windows 11 and later, enroll them in Microsoft Managed Desktop; they won't revert back to Windows 10.
```

**Line-of-business apps:** [Test Base](https://www.microsoft.com/en-us/testbase) is a resource that app publishers and IT admins can use to submit apps and test cases for Microsoft to run on a virtual machine running Windows 11 builds in a secure Azure environment. Results, test insights, and regression analysis for each test execution are available to you on a private Azure portal. Microsoft Managed Desktop will help you prioritize your line-of-business apps for validation based on app usage and reliability data. For more information about Test Base, see [Test Base for Microsoft 365](https://techcommunity.microsoft.com/t5/windows-it-pro-blog/test-base-for-microsoft-365-microsoft-ignite-2021-updates/ba-p/2185566).

### Reactive measures

> If you encounter app compatibility issues in test or production environments, you can receive no-cost support by opening a [service request] (https://docs.microsoft.com/en-us/microsoft-365/managed-desktop/working-with-managed-desktop/test-win11-mmd?view=o365-worldwide#report-issues). For Windows 11, this includes any functionality with Office, Microsoft Edge, Teams, and line-of-business applications running on the latest operating system builds. Microsoft App Assure directly engages app publishers to prioritize and resolve app compatibility issues when needed.

Reviewer: ```suggestion
If you encounter app compatibility issues in test or production environments, you can receive no-cost support by opening a [service request](/microsoft-365/managed-desktop/working-with-managed-desktop/test-win11-mmd?view=o365-worldwide#report-issues). For Windows 11, this includes any functionality with Office, Microsoft Edge, Teams, and line-of-business applications running on the latest operating system builds. Microsoft App Assure directly engages app publishers to prioritize and resolve app compatibility issues when needed.
```
